gulp/sass.js

```diff
@@ -8,9 +8,19 @@
 const gulp = require('gulp')
 const sass = require('gulp-sass')
 const sourcemaps = require('gulp-sourcemaps')
+const path = require('path')
+const fs = require('fs')
 
+const extensions = require('../lib/extensions')
 const config = require('./config.json')
 
+gulp.task('sass-plugins', function (done) {
```

Comment:
Is there a difference between 'plugins' and 'extensions', or should we just use one word throughout?

Reply:
One word throughout - I'll make the change to extensions.